source/user-manual/reference/internal-options.rst

```diff
@@ -639,6 +639,17 @@ Logcollector
 +                                          +               +----------------------------------------------------------------------------+
 |                                          |               | Any other integer between 1 and 3600.                                      |
 +------------------------------------------+---------------+----------------------------------------------------------------------------+
+| **logcollector.ip_update_interval**      | Description   | IP refresh interval, in seconds. This specifies how often the environment  |
+|                                          |               | IP is obtained to be used by the output format.                            |
+|                                          |               | .. versionadded:: 4.3                                                      |
++                                          +---------------+----------------------------------------------------------------------------+
+|                                          | Default value | 60                                                                         |
++                                          +---------------+----------------------------------------------------------------------------+
+|                                          | Allowed values| 0: It allows to obtain the IP in real-time. Systems with large             |
+|                                          |               | routing tablescan cause a heavy CPU USAGE in wazuh-modulesd.               |
++                                          +               +----------------------------------------------------------------------------+
+|                                          |               | Any other integer between 1 and 3600.                                      |
++------------------------------------------+---------------+----------------------------------------------------------------------------+
```

Comment:
I think that the next suggestions will be useful
- Link to out_format option
- Warning block
Output preview:
![Screenshot from 2021-09-16 10-08-21](https://user-images.githubusercontent.com/1791430/133617788-8e6727d3-8966-46b0-839d-3618650571db.png)

```suggestion
+------------------------------------------+---------------+----------------------------------------------------------------------------+
| **logcollector.ip_update_interval**      | Description   | IP update interval, in seconds. This specifies how often the system        |
|                                          |               | IP is obtained when the                                                    |
|                                          |               | :ref:`out_format<ossec_localfile_out_format>` option is used.              |
|                                          |               |                                                                            |
|                                          |               | .. versionadded:: 4.3                                                      |
+                                          +---------------+----------------------------------------------------------------------------+
|                                          | Default value | 60                                                                         |
+                                          +---------------+----------------------------------------------------------------------------+
|                                          | Allowed values| 0: Obtain the IP on demand.                                                |
|                                          |               |                                                                            |
|                                          |               | .. warning:: Systems with extensive routing tables                         |
|                                          |               |    can suffer from high CPU usage                                          |
+                                          +               +----------------------------------------------------------------------------+
|                                          |               | Any other integer between 1 and 3600.                                      |
+------------------------------------------+---------------+----------------------------------------------------------------------------+
```